runtime/doc/api.txt

```diff
@@ -2534,9 +2534,9 @@ nvim_buf_get_extmarks({buffer}, {ns_id}, {start}, {end}, {opts})
 
     Region can be given as (row,col) tuples, or valid extmark ids (whose
     positions define the bounds). 0 and -1 are understood as (0,0) and (-1,-1)
-    respectively, thus the following are equivalent: >lua
-      nvim_buf_get_extmarks(0, my_ns, 0, -1, {})
-      nvim_buf_get_extmarks(0, my_ns, [0,0], [-1,-1], {})
+    respectively, thus the following are equivalent: >vim
+      nvim_buf_get_extmarks(0, my_ns, 0, -1, [])
```

Comment:
do we parse "bare" calls in vim correctly now? otherwise `call` would be needed in front (alternatively, correct the existing [] the other way to lua)

Reply:
At least these do not show up as `ERROR` nodes in the tree. (The issue was the mixed syntax with `[]` in Lua.) I'm fine turning these into `vim.api` Lua examples -- just say the word!